.github/workflows/style.yaml

```diff
@@ -85,3 +91,38 @@ jobs:
             quit(status = length(problems))
           }
         shell: Rscript {0}
+        continue-on-error: true
+
+      - name: Styler check summary 🅾
+        run: |
+          cd ${{ github.event.repository.name }}
+          git status -s
+          if [[ -n `git status -s | grep .R` ]]
+          then {
+            echo "🙈 Code style is not correct!"
+            echo "🔀 The following differences were noted:"
+            git diff
+            if [ "${{ inputs.auto-update }}" == "true" ]
+            then {
+              echo "Regenerating R code files via auto-update"
+              git config --global user.name 'Insights Engineering Bot'
+              git config --global user.email '68416928+insights-engineering-bot@users.noreply.github.com'
+              echo "Adding changes to git"
+              git status -s | grep -E "\.R$|\.Rmd$|\.Rnw$|\.Rmarkdown$" | cut -f3 -d ' '  | xargs git add -v
+              git commit -am "code style - update code"
+              git push -v origin HEAD:${GITHUB_HEAD_REF}
+              echo "Pushed"
+            }
+            else {
+              echo -e "\n💻 Please rerun the following command on your workstation and push your changes"
+              echo "--------------------------------------------------------------------"
+              echo "styler::style_file(changed_r_files)"
```

Comment:
I didn't ask you to literally do this, you need to get the list of `changed_r_files` and ask the user to run that command. You need to give the user the complete list of changed files as a vector.